profile/README.md

```diff
@@ -21,17 +21,21 @@ https://github.com/jenkinsci/
 
 ### Jenkins Contributor Awards 2023
 
-The annual Jenkins awards is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+The annual Jenkins awards are happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+Vote now by filling out [this Google form](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)!
 
-Nomination categories are:
-- Most Valuable Jenkins Advocate → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6035)
-- Most Valuable Jenkins Contributor → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6033)
-- Jenkins Security MVP → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6034)
+Nominated candidates are:
+- [Most Valuable Jenkins Advocate](https://github.com/jenkins-infra/jenkins.io/issues/6035) → [Alex Earl](https://github.com/slide), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Jean-Marc Meessen](https://github.com/jmMeessen), [Kris Stern](https://github.com/krisstern), and [Mark Waite](https://github.com/MarkEWaite)
+[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
+- [Most Valuable Jenkins Contributor ](https://github.com/jenkins-infra/jenkins.io/issues/6033)→ [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal), and [Jan Faracik](https://github.com/janfaracik)
+[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
+- [Jenkins Security MVP](https://github.com/jenkins-infra/jenkins.io/issues/6034) → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB), and [Valdes Che Zogou](https://github.com/ValdesChe)
+[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
```

Comment:
```suggestion
- [Most Valuable Jenkins Advocate](https://github.com/jenkins-infra/jenkins.io/issues/6035) → [Alex Earl](https://github.com/slide), [Alexander Brandes](https://github.com/NotMyFault), [Bruno Verachten](https://github.com/gounthar), [Jean-Marc Meessen](https://github.com/jmMeessen), [Kris Stern](https://github.com/krisstern) and [Mark Waite](https://github.com/MarkEWaite): 
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
- [Most Valuable Jenkins Contributor ](https://github.com/jenkins-infra/jenkins.io/issues/6033)→ [Alexander Brandes](https://github.com/NotMyFault), [Damien Duportal](https://github.com/dduportal) and [Jan Faracik](https://github.com/janfaracik): 
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
- [Jenkins Security MVP](https://github.com/jenkins-infra/jenkins.io/issues/6034) → [Damien Duportal](https://github.com/dduportal), [Daniel Beck](https://github.com/daniel-beck), [Devin Nusbaum](https://github.com/dwnusbaum), [Kevin Guerroudj](https://github.com/Kevin-CB) and [Valdes Che Zogou](https://github.com/ValdesChe): 
[Voting is open](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)
```
Another suggestion, adding `: ` as the voting link are right after the last nominated and not on their own line

Reply:
That makes sense, I've added that to my edits!